verilog/tools/ls/symbol-table-handler.h

```diff
@@ -48,6 +48,20 @@ class SymbolTableHandler {
   // Once the project's root is set, a new SymbolTable is created.
   void SetProject(const std::shared_ptr<VerilogProject> &project);
 
+  // prepares structures for symbol-based requests
+  void Prepare();
+
+  // Returns the token (symbol) of the file at a given position, specified
+  // in the TextDocumentPositionParams
+  absl::string_view GetTokenAtTextDocumentPosition(
+      const verible::lsp::TextDocumentPositionParams &params,
+      const verilog::BufferTrackerContainer &parsed_buffers);
+
+  // Returns the location of the symbol from SymbolInfo in a
+  // Location JSON format
```

Comment:
Location is not yet json format, it is a `verible::lsp::Location` struct. So probably a good description could be  `... as a lsp::Location`

`symbol` is a bit of an overloaded term (often used as verible::Symbol*), so maybe `symbol_name` ?